### PR TITLE
Convert `value` with `format` in the print function of `Printable`

### LINKED ISSUE #120: printline function ignores format settings when printing numeric values
I think the error is in the **printline** function of **ospsuite.utils**:

https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/blob/409d2382eb49f04e9e332a126452b89b0cc2a6e5/R/printable.R#L40-L42

It just prints the value via `cat`. 
But it should check first if the value is numeric and e.g. convert it to string using the `format `function. 
Then the output would be as expected.

```
> format(dataSet$LLOQ)
[1] "1"
```

_Originally posted by @Yuri05 in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/1123#issuecomment-1249565191_

### PR DESCRIPTION
**Pull request by PavelBal**
Fixes #120

### CODE REVIEW

File: tests/testthat/_snaps/Printable.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
       x
     Output
       myPrintable: 
-         x: 
-         y: 
+         x: NULL
```
Comment (PavelBal) [inline tests/testthat/_snaps/Printable.md:7]: @Yuri05 This is another side effect. Values that are `NULL` were note visible when printing, now a `NULL` is shown.

File: NEWS.md [hunk 2]
```diff
@@ -5,6 +5,16 @@ NEW FUNCTIONS
 * `logSafe()` to compute logarithm of values that could be close to 0 or slightly 
 negative.
 
+* `foldSafe()` to compute `x / y` when `y` could be negative. All values below a
```
Comment (Yuri05) [inline NEWS.md:8]: "when x or y could be negative or zero"?

File: tests/testthat/_snaps/Printable.md (hunk 1 above)
Comment (Yuri05) [reply tests/testthat/_snaps/Printable.md:7]: nice!

File: NEWS.md (hunk 2 above)
Comment (PavelBal) [reply NEWS.md]: True, added.

File: NEWS.md [hunk 3]
```diff
@@ -5,6 +5,16 @@ NEW FUNCTIONS
 * `logSafe()` to compute logarithm of values that could be close to 0 or slightly 
 negative.
 
+* `foldSafe()` to compute `x / y` when `y` could be negative or zero. All values below a
```
Comment (Yuri05) [inline NEWS.md:8]: "**x or** y" :)

Comment (PavelBal) [reply NEWS.md]: yepp

**Review by Yuri05 — APPROVED** [review]

File: NEWS.md [hunk 4]
```diff
@@ -5,6 +5,16 @@ NEW FUNCTIONS
 * `logSafe()` to compute logarithm of values that could be close to 0 or slightly 
 negative.
 
+* `foldSafe()` to compute `x / y` when `x` or `y` could be negative or zero. All values below a 
+certain threshold `epsilon` are substituted by `epsilon`. NOTE: not suited for 
+calculating fold differences of negative numbers.
```
Comment (Yuri05) [inline NEWS.md:10]: Theoretically, we could add an optional boolean argument telling if the compared values are supposed to be negative.
If so, we would replace all values which are `>-epsilon` with `-epsilon`.
But I am not aware of any practical use case for this :)

**Pull request merged**